src/test/java/api/item/ItemLastCheckInTests.java

```diff
@@ -0,0 +1,132 @@
+package api.item;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.CoreMatchers.nullValue;
+import static org.junit.Assert.assertThat;
+
+import java.net.MalformedURLException;
+import java.util.UUID;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeoutException;
+
+import api.support.APITestContext;
+import api.support.APITests;
+import io.vertx.core.json.JsonObject;
+import org.folio.circulation.support.http.client.IndividualResource;
+import org.joda.time.DateTime;
+import org.joda.time.DateTimeZone;
+import org.junit.Before;
+import org.junit.Test;
+
+public class ItemLastCheckInTests extends APITests {
+
+  @Before public void setUp() {
+    APITestContext.defaultUserId();
```

Comment:
Should this be done after rather than before? What if the last test executed is one that changes the default and it fails?

Reply:
Done.